src/main/java/software/amazon/awssdk/crt/mqtt/OnConnectionSuccessReturn.java

```diff
@@ -0,0 +1,27 @@
+package software.amazon.awssdk.crt.mqtt;
+
+/**
+ * The data returned when the connection success callback is invoked in a connection.
+ * @see software.amazon.awssdk.crt.mqtt.MqttClientConnectionEvents
+ */
+public class OnConnectionSuccessReturn {
```

Comment:
Note: Java is the only binding that does not expose the ReturnCode from the server when a successful connection is made. However, this return code has not been exposed at all in MQTT311 for Java, while the others have exposed it prior in some form.

Because of this, I haven't exposed it here, but should we add support for it?

Reply:
No, I don't think it's really required to be honest, I just noticed it was a difference. On a success, it should always just be the same result (SUCCESS) since I do not think its possible to get a non-success code.

Thanks for taking a look at the PR!